infra/profile.py

```diff
@@ -92,8 +92,11 @@ def bench(benchmark):
 def get_llvm(runMethod, benchmark):
   path = f'./tmp/bench/{benchmark}/llvm-{runMethod}/{benchmark}-{runMethod}.ll'
 
-  with open(path) as f:
-    return f.read()
+  try:
```

Comment:
Shouldn't we return an error here or something?

Reply:
I think it's better to handle errors gracefully when possible- I wouldn't want the whole nightly run to crash if one benchmark doesn't output the llvm file. I think it's better to return `''` which we can detect on the frontend and show like "hey, this benchmark is missing the llvm"

Reply:
Ah yes I agree, I just want to make sure we write the code that detects the missing llvm and throws an error